corgidrp/data.py

```diff
@@ -763,21 +773,27 @@ def __init__(self, data_or_filepath, pri_hdr=None, ext_hdr=None, input_dataset=N
         nonlin_raw = self.data
         if nonlin_raw.ndim < 2 or nonlin_raw.shape[0] < 2 or \
         nonlin_raw.shape[1] < 2:
-            raise ValueError('The non-linearity calibration array must be at least 2x2 (room for x '
-                                'and y axes and one data point)')
+            raise ValueError('The non-linearity calibration array must be at' 
+                             'least 2x2 (room for x and y axes and one data' 
+                             'point)')
         if not np.isnan(nonlin_raw[0, 0]):
-            raise ValueError('The first value of the non-linearity calibration array  (upper left) must be set to '
-                                '"nan"')
+            raise ValueError('The first value of the non-linearity calibration '
+                             'array  (upper left) must be set to "nan"')
 
 
         # additional bookkeeping for a calibration file
         # if this is a new calibration file, we need to bookkeep it in the header
-        # b/c of logic in the super.__init__, we just need to check this to see if it is a new NonLinearityCalibration file
+        # b/c of logic in the super.__init__, we just need to check this to see if 
+        # it is a new NonLinearityCalibration file
         if ext_hdr is not None:
             if input_dataset is None:
                 # error check. this is required in this case
-                raise ValueError("This appears to be a new Non Linearity Correction. The dataset of input files needs to be passed in to the input_dataset keyword to record history of this calibration file.")
-            self.ext_hdr['DATATYPE'] = 'NonLinearityCalibration' # corgidrp specific keyword for saving to disk
+                raise ValueError("This appears to be a new Non Linearity "
```

Comment:
Just FYI, looks like you have an IDE linter at work on your computer (an example here, imposing a line length limit).  I don't think it affected too many things on this particular PR.

Reply:
very weird, but good for me to keep an eye on in the future. It's possible it was me...